### ISSUE TITLE
In multi-gpu training, occasionally the loss computed over the gpus are concatenated instead of being averaged

### ISSUE THREAD

**Issue by hdjang** [xingyizhou/CenterNet#165]
Hi, Thanks for sharing your great work :)

I have question regarding the training phase.

For the ctdet in multi-gpu training, I found that occasionally the loss in loss_stats has elements of the number of gpus, i.e., the losses computed on different gpus are concatenated instead of averaging, and this makes error in logging the training loss. Is it normal? and How can I handle this?

**Comment by xingyizhou** [comment]
We used the mean loss across GPUs https://github.com/xingyizhou/CenterNet/blob/master/src/lib/trains/base_trainer.py#L83.
